### PR TITLE
COMMERCE-2518 Minium login fixed

### PR DESCRIPTION
**Pull request by gianmarcobrunialti**

### CODE REVIEW

**Comment by gianmarcobrunialti** [comment]
ci:test

**Comment by marco-leo** [comment]
ci:test

File: commerce-theme-minium/commerce-theme-minium/src/css/_login.scss [hunk 1]
```diff
@@ -83,7 +87,10 @@
     }
 
     .minium-content {
-        height: 100%;
+        position: absolute;
+        top: 0;
+        left: 0;
+        height: 100vh;
         max-width: 100%;
```
Comment (ethanbustad) [inline commerce-theme-minium/commerce-theme-minium/src/css/_login.scss:94]: alpha sort, please!

File: commerce-theme-minium/commerce-theme-minium/src/css/_login.scss [hunk 2]
```diff
@@ -168,17 +175,32 @@
...
+                    font-size: 15px;
+                    font-weight: 500;
+                    letter-spacing: 1px;
+                }
+
+                &[type="password"] {
                     height: 55px;
-                    font-size: 33px;
+                    font-size: 50px;
+                    letter-spacing: -4px;
```
Comment (ethanbustad) [inline commerce-theme-minium/commerce-theme-minium/src/css/_login.scss:195]: this also could use an alpha sort, though it was already wrong before these changes

**Pull request merged**